src/Collections-Abstract/Collection.class.st

```diff
@@ -1183,6 +1183,16 @@ Collection >> select: aBlock [
 	^newCollection
 ]
 
+{ #category : #enumerating }
+Collection >> select: aBlock as: aClass [
```

Comment:
This method deserves a test

Reply:
I added a simple one.
`CollectionTest>>#testSelectAs:`
```Smalltalk
testSelectAs
	| collection result |
	collection := OrderedCollection new add: 1; add: 3; add: 2; yourself.
	result  := collection select: [:i | i even not] as: Array.
	self assert: result class equals: Array.
	self assert: result equals: #(1 3).
```